dashboard/15-final/package.json

```diff
@@ -31,7 +31,8 @@
   },
   "devDependencies": {
     "@types/bcrypt": "^5.0.0",
-    "dotenv": "^16.3.1"
+    "dotenv": "^16.3.1",
+    "prettier": "^3.0.3"
```

Comment:
I had to install prettier in order to format correctly 🤔 I suppose you don't have that problem? Might be my setup somehow

Reply:
Prettier is controlled from the root of next-learn, you might have to clone the root and run `pnpm i`.